client/src/components/pages/Card.jsx

```diff
@@ -3,10 +3,10 @@ import { Link } from 'react-router-dom';
 import Truncate from 'react-truncate';
 import TextTruncate from 'react-text-truncate';
 
-const Card = ({name, description, id}) => (
+const Card = ({name, description, id, image}) => (
         <div className="card" style={{ overflow: 'visible' }}>
         <div className="card-image waves-effect waves-block waves-light">
-          <img className="activator" id="activator" src={require('../../../public/images/bu.jpg')} />
+          <img className="activator" id="activator" src={image} />
```

Comment:
Good work on changing the source of the image to a prop. This makes the code cleaner and easier to read since all the details of the business are coming from the parent container. This makes your images dynamic for different businesses.